bin/node/cli/tests/running_the_node_and_interrupt.rs

```diff
@@ -28,6 +28,9 @@ use tempfile::tempdir;
 pub mod common;
 
 #[tokio::test]
+#[cfg(disable_flaky)]
+#[allow(dead_code)]
+// FIXME: https://github.com/paritytech/substrate/issues/11321
```

Comment:
This should also be fixed with my latest pr or have you seen it failing since then?

Reply:
Yeah please reenable it.

Reply:
If you also have not seen the other ones failing recently, we should also not disable them

Reply:
Oaky just re-checked: `returns_status_for_pruned_blocks` last failed two days ago, so also diabling.  
The other two ones are re-enabled.